Builds/VisualStudio2017/README.md

```diff
@@ -0,0 +1,260 @@
+# Visual Studio 2017 Build Instructions
+
+## Important
+
+We do not recommend Windows for rippled production use at this time. Currently,
+the Ubuntu platform has received the highest level of quality assurance,
+testing, and support. Additionally, 32-bit Windows versions are not supported.
+
+## Prerequisites
+
+To clone the source code repository, create branches for inspection or
+modification, build rippled under Visual Studio, and run the unit tests you will
+need these software components:
+
+* [Visual Studio 2017](README.md#install-visual-studio-2017)
+* [Git for Windows](README.md#install-git-for-windows)
+* [Google Protocol Buffers
+  Compiler](README.md#install-google-protocol-buffers-compiler)
+* [OpenSSL Library](README.md#install-openssl)
+* [Boost library](README.md#build-boost)
+* (Optional) [Cmake for Windows](README.md#optional-install-cmake-for-windows) -
+  Only needed if not using the integrated CMake in VS 2017.
```

Comment:
Not sure if it is worth mentioning that having Cmake for Windows will allow you to create VS project/solution files.